src/Task2.js

```diff
@@ -3,7 +3,7 @@ import {useState} from "react";
 
 export default function Task2() {
 
-    const [isFocus, setFocus] = useState(false);
+    const [setFocus] = useState(false);
```

Comment:
Неправильная диструктуризация, если нужно получить метод, то достаем его 

```suggestion
    const [, setFocus] = useState(false);
```